src/lib/input/autosize.ts

```diff
@@ -150,14 +150,21 @@ export class MatTextareaAutosize implements AfterViewInit, DoCheck {
       return;
     }
 
+    const previousPlaceholder = textarea.placeholder;
```

Comment:
"previous" isn't exactly right because we're not changing it. How about `placeholderText`?

Reply:
Done.